### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by ocefpaf**
Added a caveat note and some minor text updates.

@rsignell-usgs can you please review this?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,11 +15,16 @@ For IOOS python users, the
 is the easiest way to get custom packages needed to run the IOOS system tests.
 Using Anaconda users can `conda install`, which installs not only binary
 packages for their platform, but the binary libraries they depend on.
-So easier than `pip install` and more powerful than python wheels.
-System-level installation of libraries and admin privileges are not requires.
-Check out
+So easier than `pip install` and, thanks to Binary Relocation, more powerful
+than python wheels.  System-level installation of libraries and admin
+privileges are not requires. Check out
```
Comment (rsignell-usgs) [inline README.md]: should be "not required"

File: README.md [hunk 2]
```diff
@@ -15,11 +15,16 @@ For IOOS python users, the
...
 [Travis Oliphant's blog piece](http://technicaldiscovery.blogspot.com/2013/12/why-i-promote-conda.html) for more info.
 
-The recipes here are automatically tested using
-[Travis-CI](https://travis-ci.org/ioos/conda-recipes) with the assistance of [Obvious-CI](https://github.com/pelson/Obvious-CI/
-https://github.com/ioos/conda-recipes/blob/master/.travis.yml#L14-L31). 
+The recipes here are automatically built using
+[Travis-CI](https://travis-ci.org/ioos/conda-recipes) with the assistance of
+[Obvious-CI](https://github.com/pelson/Obvious-CI/https://github.com/ioos/conda-recipes/blob/master/.travis.yml#L14-L31).
+
+**CAVEAT:**  Windows users must be aware that the packages here are not fully
```
Comment (rsignell-usgs) [inline README.md]: Do we really need this caveat about testing?  I can test on windows, no? 
 I'd just end with "raise and issue" and get rid of that upstream link.

**Comment by ocefpaf** [comment]
How about?

> CAVEAT:  Some bugs, like hard-coded paths, might surface when trying to use these packages.  In that case raise an issue.

**Comment by rsignell-usgs** [comment]
Fine.

**Comment by ocefpaf** [comment]
Wrong button...

**Pull request merged**